include/glow/Graph/Graph.h

```diff
@@ -251,6 +251,12 @@ class Function final : public Named {
   Node *createChannelShuffle(llvm::StringRef name, NodeValue input,
                              size_t group, size_t kernel);
 
+  /// Removes single-dimensional entries from the shape of a tensor. The
+  /// parameter axes is a list of positive integeres, indicating the dimensions
+  /// to squeeze
+  Node *createSqueeze(llvm::StringRef name, NodeValue input,
```

Comment:
Can we add a URL to external documentation that describes the "squeeze" operation?